src/Page/Food/Builder.elm

```diff
@@ -590,50 +591,80 @@ updateIngredientFormView { excluded, db, ingredient, impact, transportImpact } =
             , onClick <| DeleteIngredient ingredientQuery
             ]
             [ Icon.trash ]
-        , span [ class "text-muted IngredientTransportLabel fs-7" ]
-            [ text "Transport pour cet ingrédient"
-            , if ingredient.planeTransport /= Ingredient.PlaneNotApplicable then
-                label
-                    [ class "PlaneCheckbox ps-2" ]
-                    [ text "("
-                    , input
-                        [ type_ "checkbox"
-                        , class "form-check-input no-outline"
+        , displayTransportDistances db ingredient ingredientQuery event
+        , span
+            [ class "text-muted text-end IngredientTransportImpact fs-7"
```

Comment:
btw, please apply the `fs-7` class to the element where `ImpactDisplay` is used, so we avoid display issue for larger numbers

![image](https://user-images.githubusercontent.com/41547/221631272-f2bfab47-36f4-4c46-a4f5-97e619d9fcdf.png)
![image](https://user-images.githubusercontent.com/41547/221631299-5a895681-6cd3-436f-a0fd-6bb6119da8e8.png)